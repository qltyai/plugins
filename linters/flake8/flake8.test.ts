import { FixtureTarget, linterCheckTest } from "tests";

const getTargetCases = (): [FixtureTarget] => {
  return [{ fixtureName: "basic.py", linterVersions: ["6.0.0"] }];
};

linterCheckTest("flake8", getTargetCases());