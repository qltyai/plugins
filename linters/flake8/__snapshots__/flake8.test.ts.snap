// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing flake8  v1 1`] = `
{
  "issues": [
    {
      "driver_name": "lint",
      "issue": {
        "level": "MEDIUM",
        "message": "'foo' imported but unused",
        "path": "foo.py",
        "plugin": "flake8",
        "ruleKey": "F401",
        "startColumn": 1,
        "startLine": 1,
      },
      "plugin_name": "flake8",
    },
  ],
}
`;
